src/alarm_req_listener.cpp

```diff
@@ -238,11 +230,11 @@ void AlarmReqListener::listener()
     // section 5.4: https://tools.ietf.org/html/rfc3877#section-5.4
     if ((msg[0].compare("issue-alarm") == 0) && (msg.size() == 3))
     {
-      AlarmTrapSender::get_instance().issue_alarm(msg[1], msg[2]);
+      _alarm_heap->issue_alarm(msg[1], msg[2]);
```

Comment:
Hmm...conceptually the alarm heap *doesn't* issue alarms. Should this be "schedule_alarm"?